template/ios/HelloWorld.xcodeproj/project.pbxproj

```diff
@@ -501,7 +495,6 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				CURRENT_PROJECT_VERSION = 1;
-				DEAD_CODE_STRIPPING = NO;
```

Comment:
This setting was introduced with [the initial import](https://github.com/facebook/react-native/commit/b74da13a75baa1345f72aebceee226f23b8ba9fe#diff-065bc63a1a0283f31962d4ab2316087bR655) of this project. The commit doesn’t provide any context as to whether this setting is needed for a FB/RN specific reason, but this is not the default setting in default Apple templates. (I tried this with both new Swift and ObjC apps.)